src/main/java/nextstep/fp/StreamStudy.java

```diff
@@ -23,11 +24,17 @@ public static long countWords() throws IOException {
     }
 
     public static void printLongestWordTop100() throws IOException {
-        String contents = new String(Files.readAllBytes(Paths
-                .get("src/main/resources/fp/war-and-peace.txt")), StandardCharsets.UTF_8);
+        String contents = new String(Files.readAllBytes(
+            Paths.get("src/main/resources/fp/war-and-peace.txt")), StandardCharsets.UTF_8
+        );
         List<String> words = Arrays.asList(contents.split("[\\P{L}]+"));
 
-        // TODO 이 부분에 구현한다.
+        words.stream()
+            .filter(word -> word.length() > 12)
+            .distinct()
+            .sorted((a, b) -> b.length() - a.length())
```

Comment:
```suggestion
            .sorted(Comparator.comparing(String::length).reversed())
```
메소드 레퍼런스를 사용하면 이렇게도 만들 수 있습니다. :)